book/tutorials/index.md

```diff
@@ -2,9 +2,7 @@
 
 Hackweek tutorials are learning-oriented and guide participants through a step-wise process with a meaningful outcome. If you are putting together a new tutorial for this event, refer to our [Hackweek Guidebook](https://guidebook.hackweek.io/training/tutorials/index.html).
 
-Below you'll find a table keeping track of all tutorials presented at this event:
-
-| Tutorial | Topics | Datasets |  Recording Link |
-| -  | - | - |  - |
-| | | | |
+Below you'll find a list of all tutorials presented at this event:
 
+```{tableofcontents}
```

Comment:
This is neat! but I like the table formatting better :) If going back to the table, adding 'Authors' could be nice for more prominent attribution... Maybe at the expense of 'topics' and 'datasets' given the tutorial title is hopefully descriptive enough...

Reply:
I was going back and forth on this and decided for a TOC.

We do have tutorial authors on the schedule and with each tutorial. That gives two locations for recognition. Feel that for website navigation, we want to be as simple as possible and the table was not the most user friendly way. 

What do others think?

Reply:
I think I like the tabular version better as well, especially to accommodate links to the videos? Although, with the A/V issues I'm not sure how many we ended up with...

The TOC approach feels too text heavy. If I want to drill down into specific sub-sections I typically do that in the menus on the left. 

Eventually it would be great to work on a curated "textbook" of all the tutorials by mission, so people don't have to search through books from different years.